packages/client/src/lib/interface.ts

```diff
@@ -46,6 +46,14 @@ export interface API {
    * Stores a single file and returns a corresponding CID.
    */
   storeBlob(service: Service, content: Blob | File): Promise<CIDString>
+  /**
+   * Stores CAR file and returns a corresponding CID.
+   */
+  storeCar(
+    service: Service,
+    content: Blob | File,
```

Comment:
```suggestion
    content: Blob |CarReader,
```

and carReader needs to be imported

Reply:
is?

content: Blob | File | CarReader,

in the case where you upload a CAR from a file input, you'd have a File. Though as File implements Blob, i'm not sure what benefit we get from stating that either is acceptable.

Reply:
yeah, just edited

Reply:
> in the case where you upload a CAR from a file input, you'd have a File. Though as File implements Blob, i'm not sure what benefit we get from stating that either is acceptable.

Yes, I think we can drop File

Reply:
`File` helps _us_ if we want to access properties on content that only `File` implements. I think it doesn't matter for consumers.

Reply:
We don't use any File properties in the current impl, so it is removed.